ParsePlatform/Docs
# Add support for Algolia DocSearch

Add a search box to each guide, powered by Algolia.

Thanks to @maxiloc for setting this up! 

## Test Plan

Built site locally and verified search functionality in each guide. Using the iOS Guide as an example,

1. Start server: `npm start`
2. Visit http://127.0.0.1:4000/docs/ios/guide/
3. Verify search box is displayed at the top of the sidebar as expected.
4. Confirm search box remains pinned at the top of the sidebar as user scrolls down.
5. Confirm results show up as user types "object", "saving", and "query".
6. Confirm tapping on search result leads to correct page.
7. Confirm title at the top is displayed properly.
8. Confirm platform color is displayed properly.
